content/docs/dvclive/usage.md

```diff
@@ -0,0 +1,143 @@
+# Usage
```

Comment:
Should we call it
```suggestion
# Usage Guide
```
? Since it's not much of a spec (except the very last part)